app/src/main/res/layout/fragment_profile_page.xml

```diff
@@ -3,131 +3,96 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MainScreen">
+    android:layout_height="match_parent">
 
-    <TextView
-        android:id="@+id/user_info_text"
-        android:layout_width="wrap_content"
-        android:layout_height="45dp"
-        android:layout_marginStart="85dp"
-        android:layout_marginTop="52dp"
-        android:layout_marginEnd="86dp"
-        android:textAlignment="center"
-        android:textSize="28sp"
-        android:textStyle="bold"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+    <RelativeLayout
+        android:id="@+id/relativeLayout"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        app:layout_constraintTop_toTopOf="parent">
 
-    <TextView
-        android:id="@+id/user_description_text"
-        android:layout_width="wrap_content"
-        android:layout_height="71dp"
-        android:layout_marginStart="85dp"
-        android:layout_marginEnd="86dp"
-        android:textAlignment="center"
-        android:textSize="20sp"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/user_info_text" />
+        <TextView

```

Comment:
Ohhhh! Thanks, I think you may have just resolved one major problem where Android couldn't perform a single click on it! :D

Reply:
Actually when I really think about it, this check should be made in EditProfile (cap the description to a certain number of chars) so that it doesn't overlap

Reply:
yeah it would make more sense